src/wled-device.ts

```diff
@@ -18,6 +25,24 @@ export class WledDevice extends Device {
 
   private colorProperty: ColorProperty;
 
+  private liveProperty: LiveProperty;
+
+  private timerProperty: TimerProperty;
+
+  private syncProperty: SyncProperty;
+
+  private effectProperty: EffectProperty;
+
+  private paletteProperty: PaletteProperty;
+
+  private effectSpeedProperty: EffectSpeedProperty;
+
+  private effectIntensityProperty: EffectIntensityProperty;
+
+  private connected?: boolean;
+
+  private intervalMs?: number;
```

Comment:
```suggestion
  private intervalMs: number;
```

Reply:
Seems I forgot to remove this one after adding default value 1000 to constructor